tools/render-test/render-test-main.cpp

```diff
@@ -840,7 +830,11 @@ static SlangResult _setSessionPrelude(const Options& options, const char* exePat
         SLANG_RETURN_ON_FAIL(TestToolUtil::getRootPath(exePath, rootPath));
 
         String includePath;
-        SLANG_RETURN_ON_FAIL(TestToolUtil::getIncludePath(rootPath, "external/nvapi/nvHLSLExtns.h", includePath));
```

Comment:
What's this change about? Strictly speaking to test for success you need SLANG_SUCCEEDED(). That testing for equality of SLANG_OK, doesn't work on non zero success codes. Admittedly that's rarely used. 

Also this change seems to make it more verbose? Was it part of debugging?

Reply:
In debug builds, SLANG_RETURN_ON_FAIL triggers an assert that prevents further execution, which is not the desired behavior here.